.mvn/maven.config

```diff
@@ -0,0 +1 @@
+-Dtycho-version=4.0.12
```

Comment:
It's just a comment, but I find such configuration even more difficult to find than having it in the pom.xml.  What's the advantage?

Reply:
Otherwise one needs to maintain the version in `pom.xml` and `extensions.xml` see https://tycho.eclipseprojects.io/doc/latest/StructuredBuild.html